src/Accelerators/NNPA/Conversion/ZLowToLLVM/CMakeLists.txt

```diff
@@ -9,6 +9,6 @@ add_onnx_mlir_library(OMZLowToLLVM
   OMZLowOps
 
   ACCEL_INCLUDE_DIRS PRIVATE
-  ${DLC_SRC_ROOT}
-  ${DLC_BIN_ROOT}
+  ${NNPA_SRC_ROOT}
```

Comment:
This should be relative to onnx-mlir source dir

Reply:
Done. This can be removed.